docker-traceability-api/pom.xml

```diff
@@ -0,0 +1,139 @@
+    furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included in
+    all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+    THE SOFTWARE.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.jenkins-ci.plugins</groupId>
+  <artifactId>docker-traceability-api</artifactId>
+  <version>1.0-beta-5-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>CloudBees Docker Traceability API</name>
+  <description>Provides an API for Docker Traceability plugin</description>
+  <url>http://wiki.jenkins-ci.org/display/JENKINS/CloudBees+Docker+Traceability</url>
+  <inceptionYear>2015</inceptionYear>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven-release-plugin.version>2.5.1</maven-release-plugin.version>
+    <docker-java.version>1.3.0</docker-java.version>
+  </properties>
+
+  <licenses>
+    <license>
+      <name>MIT</name>
+      <url>http://opensource.org/licenses/MIT</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <build>
+    <pluginManagement>
+      <!--  override parents so we can compile with 1.7 -->
+      <plugins>
+        <plugin>
+          <!-- make sure our code doesn't have 1.8 dependencies except where we know it -->
+          <!-- we depend on docker-java that uses 1.7 even though we don't require it. -->
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>animal-sniffer-maven-plugin</artifactId>
+          <version>1.14</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>check</goal>
+              </goals>
+              <phase>test</phase>
+            </execution>
+          </executions>
+          <configuration>
+            <!-- outside <execution> so that we can run 'animal-sniffer:check' from the top without breaking anything  -->
+            <signature>
+              <groupId>org.codehaus.mojo.signature</groupId>
+              <artifactId>java17</artifactId>
+              <version>1.0</version>
+            </signature>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+
+    <plugins>
+      <plugin>
```

Comment:
I'll rework POMs later. Seems one parent POM is not enough, because I need a parent POM for shading, which should also inherit from the base jenkinsci.plugin POM